### PR TITLE
NXP-20998: Add converter ai/eps to text

### PR DESCRIPTION
**Pull request by michaelva**

### CODE REVIEW

File: nuxeo-core/nuxeo-core-convert-plugins/src/main/resources/OSGI-INF/convert-service-contrib.xml [hunk 1]
```diff
@@ -108,6 +108,33 @@
...
     </converter>
 
+    <converter class="org.nuxeo.ecm.platform.convert.plugins.CommandLineConverter" name="ai2pdf">
+      <sourceMimeType>application/postscript</sourceMimeType>
+      <sourceMimeType>application/eps</sourceMimeType>
+      <sourceMimeType>application/x-eps</sourceMimeType>
+      <sourceMimeType>image/eps</sourceMimeType>
+      <sourceMimeType>image/x-eps</sourceMimeType>
+      <sourceMimeType>application/illustrator</sourceMimeType>
+      <destinationMimeType>application/pdf</destinationMimeType>
```
Comment (michaelva) [inline nuxeo-core/nuxeo-core-convert-plugins/src/main/resources/OSGI-INF/convert-service-contrib.xml]: done :)

Comment (efge) [inline nuxeo-core/nuxeo-core-convert-plugins/src/main/resources/OSGI-INF/convert-service-contrib.xml]: There's still the converter above which is named `ai2pdf`. And the `CommandLineName` below.

**Comment by efge** [comment]
👍

**Comment by nelsonsilva** [comment]
:+1:

**Pull request merged**